### PR TITLE
Add beginning groundwork for ability to query the apiserver instead of sqlalchemy.

### PR DESCRIPTION
**Pull request by JTCunning**

### CODE REVIEW

File: src/clusto/schema.py [hunk 1]
```diff
@@ -88,6 +88,7 @@ def working_version():
 SESSION.clusto_version = None
 SESSION.clusto_user = None
 SESSION.clusto_description = None
+SESSION.clusto_api = False
```
Comment (thekad) [inline src/clusto/schema.py]: wouldn't this ignore the change above in __init__.py ?

Comment (JTCunning) [inline src/clusto/schema.py]: Seeing as how all the other `SESSION` props are defined here, I don't think so?

**Comment by thekad** [comment]
ok then :+1:

**Comment by plathrop** [comment]
LGTM

**Pull request merged**